### ISSUE TITLE
iPad Air 5 M1 16.5.1

### ISSUE THREAD

**Issue by Black-7a7** [straight-tamago/misaka#233]
My iPad cannot be used and keeps restarting!

**Comment by Dimethyl-Mercury** [comment]
Yes, when I open the misaka app it shows “try kopen”, and once I turn it on my iPad restarts immediately. Anyone knows how to fix this?
